homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import "./App.css";
+import Button from "./component/button.jsx";
+import Input from "./component/todoInput";
+
+const App = () => {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      description: "Get out of bed",
+      deadline: "2017-09-11",
+      done: true
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      deadline: "2017-09-10",
+      done: false
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      deadline: "2017-09-09",
+      done: false
+    }
+  ]);
+
+  const [description, setDescription] = useState("");
+  const [date, setDate] = useState("");
+
+  const handleInput = e => {
+    e.preventDefault();
+    e.target.type === "text"
```

Comment:
Better to not let this depend on the type of the input field. If you or another developer in the future would add another field to the form the functionality of this function might suddenly break.

Reply:
Thank you for your review. I will change my code as required.

Reply:
I have changed it to e.target.name. Thank you.